services/core-web/src/HOC/AuthenticationGuard.js

```diff
@@ -1,41 +1,30 @@
-import React, { Component } from "react";
+import React, { useEffect } from "react";
 import { bindActionCreators } from "redux";
 import { connect } from "react-redux";
 import PropTypes from "prop-types";
-import Keycloak from "keycloak-js";
+import { useKeycloak } from "@react-keycloak/web";
```

Comment:
Refactored component to functional, and the "Loading" component is moved up to Index.
The keycloak.login is here because of issues having it automatically redirect (this component is better at telling if authentication is completed, in Index there was doom loops)